src/service/poc_mobile.proto

```diff
@@ -130,6 +131,30 @@ message radio_reward_share {
   uint64 end_epoch = 6;
 }
 
+message radio_reward {
+  /// Public key of the owner of the radio
+  bytes owner_key = 1;
```

Comment:
Do we need `owner_key` for the new message?  Once we hit solana, the owner_key is no longer used.

Reply:
removed